cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,22 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io')
+    cy.get(':nth-child(2) > .nav-link').click()
+
+    cy.get('[placeholder="Email"]')
+    .type('unicorn@gmailcom')
+
+    cy.get('[placeholder="Password"]')
+    .type('12345')
+
+    cy.get('.btn')
+    .click()
+
+    cy.wait(6000)
```

Comment:
please remove this `wait()`, I used it at the lecture because there was an issue with the local site, but you don't need it for this homework, and it's actually not a good practice with to use waits in tests

Reply:
Ok, but I also had the error that "timed out"

Reply:
strange, maybe it was because of the internet connection